chill-algebird/src/main/scala/com/twitter/chill/algebird/AlgebirdSerializers.scala

```diff
@@ -95,3 +86,42 @@ class HLLMonoidSerializer extends KSerializer[HyperLogLogMonoid] {
     hllMonoids.getOrElseUpdate(bits, new HyperLogLogMonoid(bits))
   }
 }
+
+class SketchMapSerializer[K, V](implicit skmMonoid: SketchMapMonoid[K, V], valueMonoid: Monoid[V])
```

Comment:
The reason we have skmMonoid is to support old serialized versions of SketchMap produced with old version of algebird 0.4, but can be eliminated (please see the other comment about that for more info).
You know, I have tried to remove this implicit SketchMapMonoid, but nor luck, I got into the deep loophole of implicit resolution order that I do not have a solution for. I would seek for your help here, please. Thank you.